src/components/ContactUsBottom/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import contactUsBottomImgRight_1 from "../../images/child.png";
+import contactUsBottomImgRight_2 from "../../images/child_2.png";
+import contactUsBottomImgLeft from "../../images/family.png";
+import "./index.css";
+import "antd/dist/antd.css";
+
+const ContactUsBottom = () => {
+  return (
+    <div className="contactUsBottomContainer">
+      <div className="contactUsBottomContainer__bottomCurve">
+        <div className="contactUsBottomContainer__bottomCurve__topHalf">
+          <div className="contactUsBottomContainer__bottomCurve__topHalfImages">
+            <img
+              className="contactUsBottomContainer__bottomCurve__topHalfImages--mainImg"
+              alt="two children playing in the garden"
+              src={contactUsBottomImgLeft}
+            />
+
+            <img
+              className="contactUsBottomContainer__bottomCurve__topHalfImages--subImgOne"
+              alt="girl playing in the garden"
```

Comment:
capitalize. Girl playing in the garden

Reply:
and all else

Reply:
Thanks for reviewing. All alt values are capitalized.